api.planx.uk/s3/s3.test.ts

```diff
@@ -0,0 +1,196 @@
+import supertest from "supertest";
+
+import app from "../server";
+
+const mockPutObject = jest.fn(() => ({
+  promise: () => Promise.resolve()
+}))
+
+let getObjectResponse = {};
+
+const mockGetObject = jest.fn(() => ({
+  promise: () => Promise.resolve(getObjectResponse)
+}))
+
+const mockGetSignedUrl = jest.fn(() => {
+  const randomFolderName = "nanoid";
```

Comment:
should this be ` = nanoid()`?

Reply:
This was intentional, so that in the test we can do things like...

```ts
expect(res.body).toEqual({
  file_type: "text/plain",
  fileUrl: expect.stringContaining("/file/private/nanoid/modified%20key")
});
```